cni.go

```diff
@@ -65,6 +107,26 @@ func (n *cni) deleteVirtInterfaces(networkNS NetworkNamespace) error {
 	return nil
 }
 
+func (n *cni) fillEndpointsPropertiesFromScan(networkNS *NetworkNamespace) error {
+	for idx, endpoint := range networkNS.Endpoints {
+		netIface, err := getIfaceByNameFromNetNs(networkNS.NetNsPath, endpoint.NetPair.VirtIface.Name)
```

Comment:
@sboeuf it would be better to get the information using netlink call in the implementation. I think the netlink call provides all the information and some additional info https://godoc.org/github.com/vishvananda/netlink#Handle.LinkByName

Do you see an advantage using the current method.

Reply:
Done